src/sagemaker/tensorflow/estimator.py

```diff
@@ -359,7 +360,7 @@ def _validate_args(
 
         if py_version == "py2" and self._only_python_3_supported():
             msg = (
-                "Python 2 containers are only available until January 1st, 2020. "
+                "Python 2 containers are only available before January 1st, 2020. "
```

Comment:
that's not strictly true. someone can always use an outdated image if they really want Python 2. why not use the same kind wording as the other places?

Reply:
This is an error message that py2 container is not available at all. If customer is trying to use a py2 container when ``self._only_python_3_supported()``

A revisit of this line makes me think we should just say py2 containers are not available with this framework version instead of saying they are available until Jan 1st.

Reply:
yeah, that sounds good. you could probably also add the latest version that is supported by Python 2 if you want

Reply:
Done